source/release-notes.html.md

```diff
@@ -11,6 +11,12 @@ weight: 200
 
 Changes to data:
 - Remove date from Rejection schema
+- Add `declined` status for application
+- Add `offer` status for application
+- Remove `conditional_offer` and `unconditional_offer` status from application
+- Add `awaiting_provider_decision` status for application
+- Add `pending_conditions` status for application
+- Remove `meeting_conditions` status from application
```

Comment:
We should also note we've added `withdrawn` and removed `unsubmitted` and `application_complete`